### ISSUE TITLE
Compilation fails if JsonCpp is compiled as a static library

### ISSUE THREAD

**Issue by 0x1100** [OSVR/OSVR-Core#501]
Commit f5326693f5f3462775b579bc7a5a3b2aef1c0740```Fix finding JsonCpp on Ubuntu 16.04 by mapping imported target configs``` breaks the build on windows.

Proposed solution:

```patch
diff --git a/cmake/FindJsonCpp.cmake b/cmake/FindJsonCpp.cmake
index cdcf1f3..1e3fcb8 100644
--- a/cmake/FindJsonCpp.cmake
+++ b/cmake/FindJsonCpp.cmake
@@ -99,7 +99,7 @@ if(jsoncpp_FOUND)
        # part of the string to indicate if we found jsoncpp_lib_static
        if(TARGET jsoncpp_lib_static)
                list(APPEND __jsoncpp_info_string "[T]")
-               _jsoncpp_apply_map_config(jsoncpp_lib)
+               _jsoncpp_apply_map_config(jsoncpp_lib_static)
        else()
                list(APPEND __jsoncpp_info_string "[]")
        endif()
```

**Comment by Hidarii** [comment]
This was the cause of broken build on Debian as well. Fixes it nicely, thanks!

**Comment by 0x1100** [comment]
Finally fixed by 07a4a262a422a8b2b0692e4f1c08bc50fb23418c.
